feature-libs/customer-ticketing/styles/components/_customer-ticketing-list.scss

```diff
@@ -0,0 +1,147 @@
+%cx-customer-ticketing-list {
+  .cx-ticketing-list-table {
+    padding: 1.5rem 0 1.125rem 0;
+    margin-bottom: 0;
+    border-top: 1px solid var(--cx-color-light);
+    border-bottom: 1px solid var(--cx-color-light);
+
+    th {
+      padding: 1.5rem 0 1.125rem 0;
+      text-align: start;
+
+      &:last-child {
+        text-align: end;
+      }
+    }
+
+    td {
+      padding: 1.625rem 0;
+
+      a {
+        min-height: 0px;
+        min-width: 0px;
+      }
+
+      @include media-breakpoint-up(md) {
+        text-align: start;
+
+        &:last-child {
+          text-align: end;
+        }
+      }
+
+      @include media-breakpoint-down(sm) {
+        width: 100%;
+        display: flex;
+        border-width: 0;
+        padding: 0 1.25rem;
+
+        &:first-child {
+          padding-top: 1.25rem;
+        }
+
+        &:last-child {
+          padding-bottom: 1.25rem;
+        }
+      }
+    }
+  }
+
+  .cx-ticketing-list-thead-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-ticketing-list-label {
+    color: var(--cx-color-secondary);
+
+    @include media-breakpoint-up(md) {
+      padding: 0 15px 0 0;
```

Comment:
if not this causes layout problems

Reply:
Hmmm have you tried it? 

I tried removing it and even adding padding: 100px to it. It won't do anything since you're literally 'hiding' the element for desktop

Reply:
> Hmmm have you tried it?
> 
> I tried removing it and even adding padding: 100px to it. It won't do anything since you're literally 'hiding' the element for desktop